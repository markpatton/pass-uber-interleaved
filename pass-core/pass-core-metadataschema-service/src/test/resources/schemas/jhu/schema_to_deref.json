{
  "$schema": "http://example.org/metadata-schemas/schemas/jhu/schema_to_dereference",
  "$id": "http://example.org/metadata-schemas/schemas/jhu/deref",
  "copySchemaName": {"$ref": "#/$schema"},
  "title": {"$ref": "schema1.json#/x/title"},
  "x": {"$ref": "schema2.json#/x"},
  "array": {"$ref": "schema3.json#/array"},
  "complexarray": {"$ref": "schema4.json#/complexarray"},
  "k": {"$ref": "schema4.json#/h/k"}
}