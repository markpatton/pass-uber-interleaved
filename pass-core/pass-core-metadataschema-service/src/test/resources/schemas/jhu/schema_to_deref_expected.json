{
	"$schema": "http://example.org/metadata-schemas/schemas/jhu/schema_to_dereference",
	"$id": "http://example.org/metadata-schemas/schemas/jhu/deref",
	"copySchemaName": "http://example.org/metadata-schemas/schemas/jhu/schema_to_dereference",
	"title": "X",
	"x": {
		"title": "x",
		"description": "an awesome letter",
		"$comment": "displays nicely",
		"type": "letter"
	},
	"array": ["c", "d", "e"],
	"complexarray": ["e", "f", {"g": "h"}],
	"k": ["l", "m", "m'"]
}